src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -79,12 +79,12 @@ public static class RSessionEvaluationCommands {
 
         public static Task<REvaluationResult> InstalledPackages(this IRExpressionEvaluator evaluation) {
             var script = @"rtvs:::packages.installed()";
-            return evaluation.EvaluateAsync(script, REvaluationKind.Json);
+            return evaluation.EvaluateAsync(script, REvaluationKind.Json | REvaluationKind.Reentrant);
```

Comment:
This means that this eval can be interrupted by the (much longer) eval for available packages, in which case it won't complete until the other one does. I would suggest using that flag only for available packages.